tests/legacy/unit/test_clr_module.py

```diff
@@ -0,0 +1,92 @@
+import os
+from pathlib import Path
+import sys
+from unittest.mock import MagicMock, patch
+
+import pytest
+
+DOTNET_ROOT = "dummy/root/path"
+DOTNET_ROOT_PATH = Path(DOTNET_ROOT)
+DOTNETCORE2_FILE = "dummy/dotnetcore2/file"
+DOTNETCORE2_BIN = "dummy/dotnetcore2/bin"
+PYEDB_FILE = "dummy/pyedb/file"
+
+
+@pytest.fixture
+def clean_environment():
```

Comment:
Should this fixture yield at the end and then restore the original state after?  Otherwise, these tests are potentially changing the global state for other tests.

Reply:
In our current CICD way of working, this shouldn't be a problem but you are correct.
I'll update that right away